src/content.css

```diff
@@ -59,6 +59,10 @@
 .notification-actions .tooltipped:after,
 .reaction-summary-item:before,
 .reaction-summary-item:after,
+.show-outdated-button:before,
+.show-outdated-button:after,
+.hide-outdated-button:before,
+.hide-outdated-button:after,
```

Comment:
This should be added as a separate block and not inside here.

Reply:
Wait, why a separate block? Anyway I already suggested to [add `rgh-tooltip` on the elements](https://github.com/sindresorhus/refined-github/pull/821#discussion_r151888383) which is already picked up.